plugins/net_plugin/net_plugin.cpp

```diff
@@ -3762,6 +4056,30 @@ namespace eosio {
             my->chain_plug->enable_accept_transactions();
          }
 
+         //if we have certificate option that TLS must be enabled
+         if ( options.count("p2p-tls-own-certificate-file") ) {
+            auto certificate = options["p2p-tls-own-certificate-file"].as<bfs::path>();
+            auto pkey        = options["p2p-tls-private-key-file"].as<bfs::path>();
+            auto ca_cert     = options["p2p-tls-security-group-ca-file"].as<bfs::path>();
```

Comment:
Should make sure key file and CA are configured if certificate is configured and print out the error, as user can make mistakes and forgets to configure all of them.

Reply:
it is not a bug, it is feature.
we allow SSL without security groups and hence without CA.
However in that case on we may need that upon security group activation we have CA configured.
@brianjohnson5972

Reply:
I fixed error check to require both certificate file and private key file be set to the same value and also require certificate file be set if security group cca file is set